### ISSUE TITLE
Figure out where to store massive data files

### ISSUE THREAD

**Issue by bgottula** [seeing-things/non-software-issues#11]
To date we have been using Google Drive to store project documents and files but personal accounts only offer 15 GB of storage which is shared with personal data such as photos and email. We are also generating enormous raw video files during each pass (23.1 GB per minute). There are options to reduce the size of these files with cropping, eliminating empty frames, and lossless compression. But even if we are diligent about doing this post-processing after every session I think we will still need more space.

Whatever we choose I'd like for there to be One Place where all of the data generated from observing goes. A possible exception is the observing log, which could still be on a Google Doc even if the files themselves live outside of Google Drive.

Options:
- Some other cheap or free cloud storage option
- Keep the files on some personal desktop / server with big HDDs

**Comment by bgottula** [comment]
@jgottula didn't you look into Google Drive for organizations or small businesses that offered a much higher storage ceiling? What was the verdict on that?

**Comment by bgottula** [comment]
After some discussion with @jgottula we came to the following conclusions:

* As noted in description, we can drastically reduce the amount of permanent storage required by cropping, cutting, and compressing the raw imagery. For example, let's say we have a SER file with 15 minutes of raw imagery at 60 FPS full-resolution (3088 x 2076). This file would be 346.1 GB. If we cut frames with no target leaving 10 minutes remaining that drops to 230.8 GB. The ISS can be up to 50 arcseconds across, so at 0.18 arcseconds per pixel at the prime focus of the EdgeHD 11 the ISS spans up to 277.8 pixels. Let's say we keep a box centered on ISS that is 300 pixels on a side. The new file size is 3.24 GB. So even before compression we have reduced the file size by a factor of over 100x in this scenario.
** We should prioritize development of better software tools to automate this.
* Google One plans offer more storage for reasonable rates. This is probably the most convenient option since it would allow us to keep everything together in Google Drive.
